src/i_joystick.c

```diff
@@ -294,14 +308,24 @@ static int GetAxisStateGamepad(int axis, int invert, int dead_zone)
 
     if (result < dead_zone && result > -dead_zone)
     {
-        result = 0;
+        return 0;
     }
 
     if (invert)
     {
         result = -result;
     }
 
+    result *= FRACUNIT / 32768; // Want FXP number between -1 and 1
+
+    // Allows for finer control when stick deflection is small.
+    if (stick_response == STICK_RESPONSE_CUBIC)
```

Comment:
`switch` on enums so the compiler will yell at you is you add a value and forget to handle it.